tetrateci/upload_upstream_release.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+
+echo "Creating a temporary directory to download $TAG release assets"
+mkdir release-tmp
+cd release-tmp
+
+echo "Fetching the download urls for the $TAG release"
+urls=$(curl -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/istio/istio/releases/tags/$TAG | jq -r '.assets[] | .browser_download_url')
+
+for url in $urls; do
+    echo "Downloading from $url"
+    wget $url
+done
+
+PACKAGES=$(ls | grep "istio")
+
+for package in $PACKAGES; do
+    echo "Publishing $package"
+    rm -f /tmp/curl.out
+    curl -T ./$package -u$BINTRAY_USER:$API_KEY $BINTRAY_API/$TAG/$package -o /tmp/curl.out
```

Comment:
We need to cross check if the package is already pushed to bintray. If so, don't push it again.

Reply:
This is an entirely new tag, do we still need to check for that?

Reply:
Can we add that check? Am sure we would delete the tag and re-run manually. We should not be touching bintray if it's already released.

Reply:
Ahh okay